### PR TITLE
Make and display warning on editing an .obj mesh

### LINKED ISSUE #20659: Godot should warn that changes are lost on editing an .obj mesh (perhaps on other imported resources too)
**Godot version:**
3.0.x, 3.1

I suspect Material_OV was added because of this, nevertheless there's no workaround for other nodes such as gridmap and multimesh and even if there was it would be impossible to set a .obj file as local to scene for example.

### PR DESCRIPTION
**Pull request by ChristopheLY**
I displayed the button warning when the mesh .obj displayed it
It's update when you add or remove the mesh

I create a new Pr because I have any problem with git rebase (#39621)

_Bugsquad edit: Originally fixed #20659_

### CODE REVIEW

File: .scons_env.json [hunk 1]
```diff
@@ -0,0 +1,701 @@
+{
```
Comment (akien-mga) [inline .scons_env.json:1]: This file should not be added, it's ignored by `.gitignore`. You need to rebase on the current `master` branch and remove it.

**Comment by aaronfranke** [comment]
@ChristopheLY Is this still desired? If so, it needs to be [rebased](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase) and tested on the latest master branch. While there are no conflicts, rebasing is important so that reviewers can easily test the PR.

**Comment by reduz** [comment]
This is no longer an issue in 4.x due to the new asset workflow, may be useful for 3.x

**Comment by YuriSizov** [comment]
Thanks for your contribution. Based on the comment above, this needs to be redone for the `3.x` branch, as it is no longer relevant in `master`.

**Pull request closed without merging**